packages/reaction-orders/server/register.js

```diff
@@ -59,34 +59,41 @@ ReactionCore.registerPackage({
     label: "Order Processing",
     status: "created",
     workflow: "coreOrderWorkflow",
-    audience: ["dashboard/orders"]
+    audience: ["dashboard/orders"],
+    emailTemplates: ["orders/new"]
```

Comment:
Previously the logic just used `let tpl = `orders/${order.workflow.status}`;`, and didn't need a `emailTemplates`... do you think it's better to have to define a template for each, or to just assume that every workflow state that supports a template, does has the `emailTemplate` assignment?